vignettes/interface-options.Rmd

```diff
@@ -0,0 +1,51 @@
+
+The JavaScript / WebAssembly files are updated soon after the NBER updates the Fortran code that runs TAXSIM 35. We will then incorporate these updated JavaScript / WebAssembly files into a new version of `usincometaxes`. A benefit of this approach is that a given version of `usincometaxes` will always produce the same results when using 'wasm'. This holds because a given version of `usincometaxes` will always use the exact same JavaScript / WebAssembly files to calculate taxes. This might not always be true with 'ssh' or 'http' since the calculations are conducted on the NBER's servers. Changes to the NBER's TASXIM program running on these servers will not correlate with version changes to `usincometaxes`. Therefore, 'wasm' is a great option if reproducibility is paramount.
+
+`usincometaxes`s [changelog](https://www.shaneorr.io/r/usincometaxes/news/index.html) will note when a version changes due to an update to the JavaScript / WebAssembly files. That way, you can check if you are using the version of `usincometaxes` with the most recent JavaScript / WebAssembly files and reinstall `usincometaxes` if this is not the case. As a reminder, the following code lets you check package versions: `packageVersion("usincometaxes")`.
+
+### 2. 'ssh'
+
+The 'ssh' `interface` option to `taxsim_calculate_taxes()` connects to the TAXSIM 35 server via ssh, sends the data to the server, and retrieves the output with calculated taxes via standard output. SSH must be installed on your operating system to use this option. If you use the 'ssh' option and SSH is not installed on your computer, you will receive an informative error telling you this fact. Because of this, there is no need to check whether SSH is installed prior to using; but you can do so with the following R command: `Sys.which('ssh')`.
+
+Standard port 22 is used to connect via SSH. There is a chance that SSH port 22 is blocked via a firewall. In this case, `taxsim_calculate_taxes()` will return an error that starts: "Error: There was a problem in calculating the taxes using 'ssh'.". Note that this error could represent a variety of problems, one of which is that a connection cannot be established with the NBER server vai SSH. But, the error might also mean that problems arose when the NBER's server attempted to calculate taxes.
+
+Using 'ssh' ensures that you are working with the most up to date version of TAXSIM 35, regardless of when you installed `usincometaxes`. With 'ssh' the good news is that you will always be using the most up to date TAXSIM 35 version, the bad news is that there is an off chance you will see slightly different results through time. Thus, 'ssh' may not always provide reproducible results. Finally, 'ssh' is slower than 'wasm' since you are sending data to, and retrieving data from, a server.
+
+### 2. 'http'
```

Comment:
probably meant to be "3." here?

Reply:
Thanks for catching. Was able to correct prior to CRAN submission.